app.js

```diff
@@ -1,42 +1,35 @@
-const { Client } = require("pg");
-var express = require("express");
-var bodyParser = require("body-parser");
-var isBot = require("isbot");
-var querystring = require("querystring");
+import pkg from 'pg';
+const { Client } = pkg;
+import express from "express";
+import bodyParser from "body-parser";
+import isBot from "isbot";
+import querystring from "querystring";
+import dotenv from "dotenv";
 
-var app = express();
+dotenv.config();
+
+const connectionString = process.env.POSTGRES_CONNECTION_STRING;
+const client = new Client({
+  connectionString,
+});
+
+client.connect();
+
+const app = express();
 
 app.use(forceHttps);
 app.use(bodyParser.json());
-app.use(
-  bodyParser.urlencoded({
-    extended: true,
-  })
-);
+app.use(bodyParser.urlencoded({ extended: true }));
 
 const port = process.env.PORT || 3000;
 app.listen(port, () => {
   console.log("Hack.af is up and running on port", port);
 });
 
-require("dotenv").config();
-
-const connectionString = process.env.POSTGRES_CONNECTION_STRING; // add your postgres connection string here or replace it with something cool variable in .env
-
-const client = new Client({
-  connectionString,
-});
-
-client.connect();
-
-var cache = {};
-
-// Pinged by uptime checker at status.hackclub.com regularly
 app.get("/ping", (req, res) => {
-  res.send("pong")
-})
+  res.send("pong");
```

Comment:
[OPTIONAL] Not entirely sure if this is a good uptime indicator. Since uptime of our database is also our responsibility, the proper status endpoint should also verify that database roundtrip is possible. A more effective thing to do here would be setting up a hack.af/status in the database and have uptime check for the redirection there.

Reply:
Added to TO-DO.